src/sourcepoint.ts

```diff
@@ -155,13 +154,21 @@ export const init = (framework: Framework, pubData = {}): void => {
 		},
 	};
 
+	// NOTE - Contrary to the SourcePoint documentation, it's important that we add EITHER gdpr OR ccpa
```

Comment:
Can we link to the relevant bit of the (incorrect) Sourcepoint documentation here? It might be handy for future folk working on this code to understand the context.